README.md

```diff
@@ -225,7 +223,7 @@ In general, we’ve found it makes the most sense to test components based on th
 To test a specific variant you can wrap your test with the `with_variant` helper method as:
 
 ```ruby
-def test_render_component_for_tablet
+test "render component for table" do
```

Comment:
```suggestion
test "render component for tablet" do
```

Reply:
Well, that's debatable. If you take a look on the official documentation, you'll see some examples using this syntax.